config.yml.erb

```diff
@@ -517,7 +517,6 @@ devinternal_db_writer: !Secret
 # Temporarily set in locals.yml until Stack Secrets are ready to deploy.
 db_cluster_id:
 
-# FOLLOW ON: Transition from the above naming convention to the `db_endpoint_[type]` convention below.
```

Comment:
nit; I believe `TODO` is the convention here :smile:

Reply:
Fortunately getting rid of that with this change!